### PR TITLE
Fix performance regression detection in CI #1248

### PR DESCRIPTION
**Pull request by ToSeven**

### CODE REVIEW

File: scripts/executions-check.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    sleep 1
+
+	timeout 5m ./fuzzer_libpng > /dev/null 2>&1 &
+
+	while true; do
+		if grep -q "Broker" log.txt ; then
+			pkill -9 "fuzzer_libpng"
+			executions=$(grep  -m 1 "Broker" log.txt  | awk '{print $14}')
+			rm -rf ./libafl_unix_shmem_server
+			echo "${executions%,}"
```
Comment (domenukk) [inline scripts/executions-check.sh:33]: Should we get the value of the last run(s) and if it's far lower, fail CI?

Comment (tokatoka) [reply scripts/executions-check.sh:33]: or maybe put auto-comment about the # of executions on the github page?
something like this
https://github.com/marketplace/actions/auto-comment

Comment (ToSeven) [reply scripts/executions-check.sh:33]: `auto-comment` looks like a good idea.

Comment (domenukk) [reply scripts/executions-check.sh:33]: Please open a follow-up PR for that :)

**Pull request merged**